backend/eventsubscribers/donationEvents.js

```diff
@@ -1,18 +1,45 @@
 const EventEmitter = require('events');
 const { model } = require('../models/BankItem');
+const User = require( '../models/User' );
+require('dotenv').config();
+const stripe = require('stripe')(process.env.STRIPE_SECRET_KEY);
 
 class DonationEventsEmitter extends EventEmitter {}
 
 const donationEventsEmitter = new DonationEventsEmitter();
 
-donationEventsEmitter.on('donationAmountLimitReached', (donationRequest, totalRoundupByUsers) => {
+donationEventsEmitter.on('donationAmountLimitReached', async (donationRequest, totalRoundupByUsers) => {
   console.log('donationAmountLimitReached has been fired');
   console.log('donationRequest: ', donationRequest);
   for(const userSpecificTotalRoundup of totalRoundupByUsers) {
       console.log(`Total roundup of ${userSpecificTotalRoundup.totalRoundup.toFixed(2)} for user ObjectId("${userSpecificTotalRoundup.user_id}")`);
       // TODO: charge the user's CC
-      
-      // TODO: send donationRequest completion email notification to the user
+      const user = await User.findById(userSpecificTotalRoundup.user_id).exec();
+      const paymentMethods = await stripe.paymentMethods.list({
+        customer: user.stripeCustomerId,
+        type: 'card',
+      });
```

Comment:
```suggestion
      const customer = await stripe.customers.retrieve(user.stripeCustomerId);
```
You should get the customer object (which has the default payment method id) instead of getting a list of paymentMethods.